pkg/apis/acme/v1beta1/types_issuer.go

```diff
@@ -192,6 +193,7 @@ type ACMEChallengeSolverHTTP01Ingress struct {
 	// This is typically used in conjunction with ingress controllers like
 	// ingress-gce, which maintains a 1:1 mapping between external IPs and
 	// ingress resources.
+	// A reference by name to a Kubernetes Ingress object (see doct at https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource)
```

Comment:
```suggestion
	// A reference by name to a Kubernetes Ingress object (see docs at https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource)
```

This typo is repeated elsewhere too.